tools-local/init-build.proj

```diff
@@ -107,16 +107,15 @@
           Condition="'$(OfflineBuild)' == 'true'">
     <ItemGroup>
       <_ToolPackage
-        Include="$(SourceBuiltPackagesPath)%.1.0.0-dev.nupkg"
-        Exclude="$(SourceBuiltPackagesPath)%.1.0.0-dev.symbols.nupkg"
-        Id="%(ArcadeSdkOverride.Identity)" />
+        Include="$(PrebuiltSourceBuiltPackagesPath).Microsoft.DotNet.Arcade.1.0.0-dev.nupkg"
+        Exclude="$(PrebuiltSourceBuiltPackagesPath).Microsoft.DotNet.Arcade.1.0.0-dev.symbols.nupkg"
     </ItemGroup>
 
     <ZipFileExtractToDirectory SourceArchive="%(_ToolPackage.Identity)"
-                               DestinationDirectory="$(ToolPackageExtractDir)%(_ToolPackage.Id)\"
+                               DestinationDirectory="$(PackagesDir)microsoft.dotnet.arcade.sdk/1.0.0-dev/"
                                OverwriteDestination="true" />
     <ItemGroup>
-      <ExtractedToolFiles Include="$(ToolPackageExtractDir)%(_ToolPackage.Id)/**/*netcore*/*.dll" />
+      <ExtractedToolFiles Include="$(PackagesDir)microsoft.dotnet.arcade.sdk/1.0.0-dev/**/*netcore*/*.dll" />
     </ItemGroup>
 
     <Copy SourceFiles="@(ExtractedToolFiles)" DestinationFolder="$(ToolPackageExtractDir)/" />
```

Comment:
Is the destination folder correct?

Reply:
`$(ToolPackageExtractDir)` is `Tools/source-built`, so I think you want the package name as well: `$(ToolPackageExtractDir)Microsoft.DotNet.Arcade.Sdk/`.  There might be some [RecursiveDir](https://blogs.msdn.microsoft.com/msbuild/2005/11/07/how-to-recursively-copy-files-using-the-copy-task/) trick you need to do if that doesn't look right.

Reply:
I don't think the folder is correct.  I don't think we want $(ToolPackageExtractDir).  I think we want $(PackagesDir).  i.e. I think the destination should be "$(PackagesDir)microsoft.dotnet.arcade.sdk/1.0.0-dev/".

Reply:
The `ZipFileExtractToDirectory` call has `DestinationDirectory="$(PackagesDir)microsoft.dotnet.arcade.sdk/1.0.0-dev/"` already though right?  We want it in both the package cache and the `source-built` folder, I think.

Reply:
In that case, won't source and destination become the same?

https://github.com/adaggarwal/source-build/blob/7cbbc77254f6928866a8e8b253530e01e5c56da9/tools-local/init-build.proj#L121